lib/deprecation_toolkit/behaviors/raise.rb

```diff
@@ -22,9 +22,16 @@ class DeprecationIntroduced < DeprecationException
       def initialize(current_deprecations, recorded_deprecations)
         introduced_deprecations = current_deprecations - recorded_deprecations
 
+        record_message =
+          if defined?(RSpec)
+            "You can record deprecations by setting `config.deprecation_behavior = :record` in your RSpec config."
```

Comment:
Hey Edouard. Thanks so much for the feedback! I'm adding specs to this and working through your feedback. 

Couple thoughts worth considering:

This is probably the biggest difference to how this would work b/w RSpec / Minitest. As far as I can tell RSpec does not support adding custom flags when running a test. See [this issue](https://github.com/rspec/rspec-core/issues/2258). 

The only way I can think to do it is to add an ENV var before you run the test. Something like: 
```ruby
DEPRECATION_BEHAVIOR="raise" bundle exec rspec path/to/spec
```

Not sure if you think the gem should support this or we should just leave it up to the implementor.

Reply:
Ah yeah indeed thanks for pointing to the issue. I'd prefer to do that with an ENV variable like you described. This can also be useful outside of RSpec (on CI for example).